---
title: "A comprehensive transcriptomic meta-Analysis leveraging deep learning to uncover molecular signatures and potential therapeutic targets in Triple-Negative Breast Cancers"
authors:
- S Salesi, T Adel, Z Nayeri, V shariati, A Zomorodipour
date: "2025-08-09T00:00:00Z"
doi: ""

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Accepts a single type but formatted as a YAML list (for Hugo requirements).
# Enter a publication type from the CSL standard.
publication_types: ["article"]

# Publication name and optional abbreviated publication name.
publication: ""
publication_short: ""

abstract: Triple-negative breast cancer (TNBC), characterized by its aggressive behavior and lack of hormone receptor expression, remains a therapeutic challenge. This study integrates multi-omics data and AI-driven approaches to dissect the molecular mechanisms driving TNBC progression. Through a meta-analysis of 49 transcriptomic studies (2013–2024), we identified 2,101 differentially expressed genes (DEGs), including 68 consistently dysregulated protein-coding genes, with CXCL10 (↑4.01-fold) and ADH1B (↓4.8-fold) as the most significantly altered. Pathway enrichment revealed upregulated genes associated with cell proliferation, immune evasion, and metabolic reprogramming, while downregulated genes implicated hormonal signaling suppression and extracellular matrix remodeling. Gene Ontology analysis highlighted mitotic regulation and immune dysregulation as central processes. AI-based clustering of protein-protein interaction networks identified five functional modules (Tumor Growth, Invasion & Metastasis, Metabolism, Immune & Inflammation, Hormonal & Stress Response), with hub genes like CDK1 and CXCL8 driving tumor proliferation and immune escape. Notably, machine learning algorithms enhanced data integration and cluster identification, revealing FOXM1 as a key regulator of mitotic pathways (p = 6.189E-07) and JUN as a mediator of stromal-epithelial interactions despite its downregulation. Hormonal profiling uncovered systemic suppression of estrogen-responsive (*ESR1*, *FOXA1*) and neurohormonal (*ADRB1*, *AGTR1*) pathways, emphasizing TNBC’s endocrine-silenced phenotype. Immune analysis demonstrated dual chemokine dysregulation, loss of homeostatic signals (CXCL12, CCL21) impaired immune surveillance, while pro-inflammatory chemokines (CXCL8, CCL20) recruited immunosuppressive cells. Chronic interferon signaling (STAT1, ISG15) and immune checkpoint alterations (CTLA4, CD80/86) further shaped a resistant microenvironment. These findings elucidate TNBC’s complex molecular landscape, integrating AI-enhanced analytics to prioritize therapeutic targets, such as mitotic kinases, chemokine networks, and hormonal signaling nodes, offering actionable insights for overcoming therapy resistance in this aggressive subtype. 

# Summary. An optional shortened abstract.
summary: Triple-negative breast cancer (TNBC), characterized by its aggressive behavior and lack of hormone receptor expression, remains a therapeutic challenge. This study integrates multi-omics data and AI-driven approaches to dissect the molecular mechanisms driving TNBC progression. Through a meta-analysis of 49 transcriptomic studies (2013–2024), we identified 2,101 differentially expressed genes (DEGs), including 68 consistently dysregulated protein-coding genes, with CXCL10 (↑4.01-fold) and ADH1B (↓4.8-fold) as the most significantly altered. Pathway enrichment revealed upregulated genes associated with cell proliferation, immune evasion, and metabolic reprogramming, while downregulated genes implicated hormonal signaling suppression and extracellular matrix remodeling. Gene Ontology analysis highlighted mitotic regulation and immune dysregulation as central processes. AI-based clustering of protein-protein interaction networks identified five functional modules (Tumor Growth, Invasion & Metastasis, Metabolism, Immune & Inflammation, Hormonal & Stress Response), with hub genes like CDK1 and CXCL8 driving tumor proliferation and immune escape. Notably, machine learning algorithms enhanced data integration and cluster identification, revealing FOXM1 as a key regulator of mitotic pathways (p = 6.189E-07) and JUN as a mediator of stromal-epithelial interactions despite its downregulation.

tags:
- Cancer
- Meta
- Transcriptome
- Human
- ML
- DL
- Genome
- Epigenomics
- Network

featured: true

links:
- name: Custom Link
  url: ''
url_pdf: ''
url_code: ''
url_dataset: 'https://docs.google.com/spreadsheets/d/1aOQgYP_vS6Xf0MV6sRsRl_-vBzJQoz5I/edit?gid=577749617#gid=577749617'
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Figure 8. Clustring of metagenes'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects:
- cancer-meta-analysis
- tnbc-project

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: tnbc meta-analysis
---

