---
title: "Transcriptome meta-analysis of tumor-associated mesenchymal stem cells reveals differentially expressed genes driving tumor microenvironment remodeling and cancer progression"
authors:
- Babak Jahangiri, Vahid Shariati, Elahe Asadollahi, Zahra-Soheila Soheili, Mehdi Shamsara, Alireza Zomorodipour
date: "2025-09-27T00:00:00Z"
doi: ""

# Schedule page publish date (NOT publication's date).
publishDate: ""

# Publication type.
# Accepts a single type but formatted as a YAML list (for Hugo requirements).
# Enter a publication type from the CSL standard.
publication_types: ["article"]

# Publication name and optional abbreviated publication name.
publication: ""
publication_short: ""

abstract: Tumor-Associated Mesenchymal Stem Cells (TA-MSCs) are critical modulators of the Tumor Microenvironment (TME), and their innate tumor-homing ability is ideal for targeted immunotherapy. However, leveraging this tropism requires identifying endogenous regulatory elements that are exclusively activated within the TME to guarantee local gene expression and minimize systemic toxicity. To address this need, we implemented a robust, multi-platform transcriptome meta-analysis pipeline. The study aggregated and processed raw data from multiple human RNA-seq and microarray datasets, comparing TA-MSCs (derived from breast, gastric, ovarian, colorectal, and lung cancers) against control Mesenchymal Stem Cells (MSCs). Differentially Expressed Genes (DEGs) were identified using statistical models (DESeq2/eBayes) and filtered based on stringent criteria, requiring high and consistent upregulation (Log2 Fold Change > 1; $P_{adj} < 0.05$) across diverse TA-MSC profiles, coupled with negligible baseline expression in resting MSCs. This rigorous filtering process yielded a core panel of TME-induced genes. Among these, the top 10 genes was identified as the  most robust candidates, demonstrating the highest consistency and magnitude of induction across the majority of datasets. Subsequent RT-PCR validation confirmed that their expression was strongly and specifically induced in human bone marrow MSCs upon co-culture with triple-negative breast cancer cells. This successful bioinformatics-driven approach establishes the most reliable candidates as a highly selective molecular switch for tumor-activated immunogene therapy.

# Summary. An optional shortened abstract.
summary: Tumor-Associated Mesenchymal Stem Cells (TA-MSCs) are critical modulators of the Tumor Microenvironment (TME), and their innate tumor-homing ability is ideal for targeted immunotherapy. However, leveraging this tropism requires identifying endogenous regulatory elements that are exclusively activated within the TME to guarantee local gene expression and minimize systemic toxicity. To address this need, we implemented a robust, multi-platform transcriptome meta-analysis pipeline. The study aggregated and processed raw data from multiple human RNA-seq and microarray datasets, comparing TA-MSCs (derived from breast, gastric, ovarian, colorectal, and lung cancers) against control Mesenchymal Stem Cells (MSCs). Differentially Expressed Genes (DEGs) were identified using statistical models (DESeq2/eBayes) and filtered based on stringent criteria, requiring high and consistent upregulation (Log2 Fold Change > 1; $P_{adj} < 0.05$) across diverse TA-MSC profiles, coupled with negligible baseline expression in resting MSCs. This rigorous filtering process yielded a core panel of TME-induced genes. Among these, the top 10 genes was identified as the  most robust candidates, demonstrating the highest consistency and magnitude of induction across the majority of datasets. Subsequent RT-PCR validation confirmed that their expression was strongly and specifically induced in human bone marrow MSCs upon co-culture with triple-negative breast cancer cells. This successful bioinformatics-driven approach establishes the most reliable candidates as a highly selective molecular switch for tumor-activated immunogene therapy.

tags:
- Cancer
- Breast
- MSC
- Meta-analysis
- Human
- Transcriptome

featured: true

links:
- name: ''
  url: ''
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Figure 3. Multidimensional visualization of metagenes'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects:
- cancer-meta-analysis

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
# slides: tnbc drug discovery
---

